docs/examples/State Estimation Example.ipynb

```diff
@@ -504,6 +528,87 @@
     "display(pd.DataFrame(output_data[\"sym_load\"]))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b2c7c914",
+   "metadata": {},
+   "source": [
+    "### Calculation Method\n",
+    "\n",
+    "There are two calculation methods available and they are defined as the following enumrations:\n",
+    "* **Iterative linear method**: \n",
+    "    * `CalculationMethod.iterative_linear`: All measurements are linearized before running the method.\n",
+    "* **Newton Raphson method**: The Newton-Raphson method considers the problem as a system of real, non-linear equations. It does not make any assumptions on linearization. It could be more accurate and converge in less iterations than the iterative linear method.\n",
+    "    * `CalculationMethod.newton_raphson`: traditional Newton-Raphson method."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00be7ae5",
+   "metadata": {},
+   "source": [
+    "**Following command does state estimation with Newton Raphson method. Note the difference of result for node compared to the results from iterative linear method.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40e6d425",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_data_NR = model.calculate_state_estimation(symmetric=True, calculation_method=CalculationMethod.newton_raphson)\n",
+    "\n",
+    "print(\"Node result\")\n",
+    "display(pd.DataFrame(output_data_NR[\"node\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a90ab8fb",
+   "metadata": {},
+   "source": [
+    "In **Newton Raphson method**, standard deviations of active and reactive power mesurement errors can also be provided seperately. We refer a detailed explanation about providing standard deviations for different methods to the [Documentation](../user_manual/calculations.md). Below is a example of a state estimation with seperate standard deviations of p and q measurement.  "
```

Comment:
maybe rephrase. this statement is true in `iterative_linear` as well, but the behaviour is different.

in particular, the degeneracy of the pair `sigma_p, sigma_q` breaks whenever there are 2 sensors with different ratios between `sigma_p` and `sigma_q` and they are aggregated into a single measurement (which happens only at the node level at most)

for `newton_raphson`, it doesn't matter whether they are on the same terminal or not and it is therefore more general